test/karma.conf.js

```diff
@@ -1,33 +1,29 @@
-// Karma configuration
-// Generated on Fri Sep 27 2013 23:41:22 GMT+0200 (W. Europe Daylight Time)
-
 module.exports = function(config) {
   config.set({
 
+    plugins: [
+      'karma-phantomjs-launcher',
+      'karma-mocha',
+      'karma-chai'
+    ],
+
+    customLaunchers: {
+      Chrome_without_security: {
+        base: 'Chrome',
+        flags: ['--disable-web-security']
+      },
+      PhantomJS_without_security: {
+        base: 'PhantomJS',
+        flags: ['--web-security=no']
+      }
+    },
+
     // base path, that will be used to resolve files and exclude
     basePath: '..',
 
-
     // frameworks to use
     frameworks: ['mocha', 'chai'],
 
-
-    // list of files / patterns to load in the browser
-    files: [
-      'bower_components/jquery/dist/jquery.js',
-      'bower_components/angular/angular.js',
-      'bower_components/angular-mocks/angular-mocks.js',
-      'modules/*/*.js',
-      'modules/*/test/*.test.js'
-    ],
-
-
-    // list of files to exclude
-    exclude: [
-
-    ],
-
-
     // test results reporter to use
     // possible values: 'dots', 'progress', 'junit', 'growl', 'coverage'
     reporters: ['progress'],
```

Comment:
Even better, throw in a coverage reporter as shown in https://github.com/karma-runner/karma-coverage